### ISSUE TITLE
trtexec convert onnx failed

### ISSUE THREAD

**Issue by lsr12345** [NVIDIA/TensorRT#2181]
## Description
I use trtexec (TensorRT 8.4.1.5 GA CUDA11.4) to convert my onnx model to trt engine, It show the error below. 
Used command: ./trtexec --onnx=./model_sim.onnx  --saveEngine=model_sim.engine  --verbose
I don't know why this error happens?

[07/26/2022-15:17:33] [V] [TRT] Generating copy for 3548 to 3551 because copy elision not implemented for axis.
[07/26/2022-15:17:33] [V] [TRT] Eliminating concatenation Concat_2067
[07/26/2022-15:17:33] [V] [TRT] Retargeting 2375 to 3567
[07/26/2022-15:17:33] [V] [TRT] Retargeting 2610 to 3567
[07/26/2022-15:17:33] [V] [TRT] Retargeting 2845 to 3567
[07/26/2022-15:17:33] [V] [TRT] Retargeting 3080 to 3567
[07/26/2022-15:17:33] [V] [TRT] Retargeting 3315 to 3567
[07/26/2022-15:17:33] [V] [TRT] Retargeting 3550 to 3567
[07/26/2022-15:17:33] [V] [TRT] After concat removal: 629 layers
[07/26/2022-15:17:33] [V] [TRT] Graph construction and optimization completed in 0.769753 seconds.
[07/26/2022-15:17:34] [V] [TRT] Using cublasLt a tactic source
[07/26/2022-15:17:34] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +750, GPU +318, now: CPU 1306, GPU 716 (MiB)
[07/26/2022-15:17:34] [V] [TRT] Using cuDNN as a tactic source
[07/26/2022-15:17:34] [I] [TRT] [MemUsageChange] Init cuDNN: CPU +618, GPU +268, now: CPU 1924, GPU 984 (MiB)
[07/26/2022-15:17:34] [W] [TRT] Detected invalid timing cache, setup a local cache instead
[07/26/2022-15:17:34] [V] [TRT] Constructing optimization profile number 0 [1/1].
[07/26/2022-15:17:34] [V] [TRT] *************** Autotuning format combination: Float(48640,2560,128,1), Float(35840,1280,128,1) -> Float(3584,128,1), Float(6016,6016,128,1), Float(6016,6016,47,1), Float(6016,6016,128,1), Float(6016,6016,47,1), Float(6016,6016,128,1), Float(6016,6016,47,1), Float(6016,128,1), Float(768,128,1), Float(128,128,128,1), Float(128,128,128,1), Float(28,28,28,1) ***************
[07/26/2022-15:17:34] [V] [TRT] --------------- Timing Runner: {ForeignNode[Reshape_0...MatMul_384]} (Myelin)
[07/26/2022-15:17:34] [I] [TRT] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +0, GPU +0, now: CPU 1924, GPU 966 (MiB)
[07/26/2022-15:17:34] [E] Error[1]: [codeGenerator.cpp::createMyelinGraph::314] Error Code 1: Myelin (myelinTargetSetPropertyMemorySize called with invalid memory size (0).)
[07/26/2022-15:17:34] [E] Error[2]: [builder.cpp::buildSerializedNetwork::417] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed.)
[07/26/2022-15:17:34] [E] Engine could not be created from network
[07/26/2022-15:17:34] [E] Building engine failed
[07/26/2022-15:17:34] [E] Failed to create engine from model or file.
[07/26/2022-15:17:34] [E] Engine set up failed

## Environment

**TensorRT Version**: 8.4.1.5
**NVIDIA GPU**: 3070
**NVIDIA Driver Version**: 470.94
**CUDA Version**: 11.4
**CUDNN Version**: 
**Operating System**: ubuntu 18.04
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files
onnx file: 
https://drive.google.com/file/d/1PD1L0AKIzHd5zUik_FVWTDyATv0KYV8A/view?usp=sharing

**Comment by jackwish** [comment]
@lsr12345 Thanks for reporting this issue! I tried your model, it can successfully run on A100 and other GPU with the same driver version. Could you check your machine setup and which kind of build did you use (e.g. Ubuntu 20.04 deb or Linux tar from [this page](https://developer.nvidia.com/nvidia-tensorrt-8x-download))?

Btw, we have just released TensorRT 8.4 GA Update 1 (8.4.2), could you try that build?

**Comment by lsr12345** [comment]
Thanks  a lot for your advice, I will try it

**Comment by lsr12345** [comment]
closed

**Comment by zewenli98** [comment]
Hi, I got the same issue. How did you fix it?
```
$ trtexec --onnx=demo.onnx --verbose --dumpProfile --exportLayerInfo=resnet18_layer_info.json --exportProfile=resnet18_profile.json --profilingVerbosity=detailed --safe
...
[08/08/2022-08:15:27] [E] Engine could not be created from network
[08/08/2022-08:15:27] [E] Building engine failed
[08/08/2022-08:15:27] [E] Failed to create engine from model or file.
[08/08/2022-08:15:27] [E] Engine set up failed
```
Everything is normal if I run the command without `--safe`. I tried many ONNX models but all were failed.

**Comment by lsr12345** [comment]
just restart machine…



---Original---
From: "Zewen (Evan) ***@***.***&gt;
Date: Mon, Aug 8, 2022 23:22 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [NVIDIA/TensorRT] trtexec convert onnx failed (Issue #2181)




 
Hi, I got the same issue. How did you fix it?
 $ trtexec --onnx=demo.onnx --verbose --dumpProfile --exportLayerInfo=resnet18_layer_info.json --exportProfile=resnet18_profile.json --profilingVerbosity=detailed --safe ... [08/08/2022-08:15:27] [E] Engine could not be created from network [08/08/2022-08:15:27] [E] Building engine failed [08/08/2022-08:15:27] [E] Failed to create engine from model or file. [08/08/2022-08:15:27] [E] Engine set up failed  
Everything is normal if I run the command without --safe. I tried many ONNX models but all were failed.
